packages/botonic-react/src/webview.jsx

```diff
@@ -40,7 +40,10 @@ class App extends React.Component {
                 console.log(e)
             }
         }
-        window.WebviewSdk.close(() => {}, err => console.log(err))
+        if (this.state.session.user.provider === 'whatsappnew') {
```

Comment:
in the future, it would be nice to have constants for the provider names to avoid repeating the whatapp-gate :-)